libs/designer-ui/src/lib/tokenpicker/index.tsx

```diff
@@ -186,7 +186,7 @@ export function TokenPicker({
         }}
       >
         <div
-          className="msla-token-picker-container"
+          className="msla-token-picker-container-v3"
```

Comment:
@Eric-B-Wu - Will this impact our css class?

Reply:
I think this might not actually affect our CSS because msla-token-picker-container is not a selector actually used in our css

Reply:
Yeah, we are not using 'msla-token-picker-container' in tokenpicker.less